config/maistra.io_servicemeshmemberrolls.yaml

```diff
@@ -0,0 +1,148 @@
+
+---
```

Comment:
@rcernich what makes these?

Reply:
run `make generate-manifests`

Reply:
Ran `make generate-manifests`. All of the new files are still there though. I'm assuming that's good.